### PR TITLE
mp b2c customer access endpoint

### PR DESCRIPTION
**Pull request by himanshu-saxena-metacube**
CC-16253 customer access endpoint

### CODE REVIEW

File: tests/api/mp_b2c/glue/customer_endpoints/customer_restore_password/negative.robot [hunk 1]
```diff
@@ -0,0 +1,87 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+Test Setup        TestSetup
+Default Tags    glue
+
+*** Test Cases ***
+ENABLER
+    TestSetup
+Restore_password_without_customer_id
```
Comment (OKrivtsova) [inline tests/api/mp_b2c/glue/customer_endpoints/customer_restore_password/negative.robot:10]: please remove the unrelated changes - the password changes belong to their own PR

Comment (himanshu-saxena-metacube) [reply tests/api/mp_b2c/glue/customer_endpoints/customer_restore_password/negative.robot]: Fixed

**Review by OKrivtsova — APPROVED** [review]

**Review by OKrivtsova — COMMENTED** [review]
please merge master into your branch and fix the conflicts

**Comment by himanshu-saxena-metacube** [comment]
It seems its already merged to Master, hence closing this out for now
